components/monitoring/grafana/base/grafana-app.yaml

```diff
@@ -101,6 +101,7 @@ spec:
   config:
     users:
       viewers_can_edit: true
+      auto_assign_org_role: Editor
```

Comment:
This won't be needed IMO. The `viewers_can_edit` should grant users the ability to edit dashboards and run custom queries.

Reply:
Ok, I agree. There's an `example` dashboard that can be used as a starting point when trying to create a new dashboard. Closing this PR.